info-compiler/src/main/java/com/google/sps/webcrawler/NewsContentProcessor.java

```diff
@@ -149,7 +149,6 @@ public static NewsArticle summarize(NewsArticle originalNewsArticle) throws Exce
       vertices.add(new Vertex(new IntValue(i), new StringValue(sentences[i])));
       for (int j = i; j < sentences.length; j++) {
         if (i == j) {
-          edges.add(new Edge(new IntValue(i), new IntValue(j), new DoubleValue(0)));
           continue;
         }
```

Comment:
Instead of this, could you just start the for loop at j = i + 1?

Reply:
That makes sense! Thank you